src/views/Datasets.vue

```diff
@@ -49,6 +49,10 @@
 		<!-- alerts -->
 		<b-alert variant="danger" :show="!!error" dismissible @dismissed="error = null">{{ error }}</b-alert>
 
+		<b-alert :show="otherError" variant="danger">
```

Comment:
why this and not the other above? and why here

Reply:
I am not sure how to combine two different errors here?

Reply:
I would see that this is completely wrong place to display an error which belongs to another view.